doc/manual/manual.md

```diff
@@ -3710,16 +3710,18 @@ values in scope at the time.
 * `mir_postcond : Term -> MIRSetup ()`
 * `mir_assert : Term -> MIRSetup ()`
 
-These commands take `Term` arguments, and therefore cannot describe
-the values of pointers. The "assert" variants will work in either pre-
-or post-conditions, and are useful when defining helper functions
-that, e.g., state datastructure invariants that make sense in both
-phases.  The `llvm_equal` command states that two `SetupValue`s should
-be equal, and can be used in either the initial or the final state.
+These commands take `Term` arguments, and therefore cannot describe the values
+of pointers. The "assert" variants will work in either pre- or post-conditions,
+and are useful when defining helper functions that, e.g., state datastructure
```

Comment:
If you're changing this paragraph anyway, can you change "state" to "provide"? Otherwise it's easy to read "state" as a noun and then the whole thing becomes confusing. Not that I just did this or anything.

Reply:
Done.